src/test/java/com/example/project/page_object/CheckoutPage.java

```diff
@@ -0,0 +1,59 @@
+package com.example.project.page_object;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class CheckoutPage {
+    private WebDriver driver;
+
+    @FindBy(name = "countries")
+    private WebElement countrySelect;
+    @FindBy(name = "firstname")
+    private WebElement firstNameInput;
+    @FindBy(name = "lastname")
+    private WebElement lastNameInput;
+    @FindBy(name = "streetAddress")
+    private WebElement streetAddressInput;
+    @FindBy(name = "city")
+    private WebElement cityInput;
+    @FindBy(name = "states")
+    private WebElement statesSelect;
+    @FindBy(name = "postalCode")
+    private WebElement zipCodeInput;
+
+
+    public CheckoutPage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    public void setCountry(String country) {
```

Comment:
just a suggestion, try to make this and below methods return `CheckoutPage`, Once done you can chain these methods in the test.